### PR TITLE
Scarab Ship Fixes

### PR DESCRIPTION
**Pull request by RustingWithYou**
Makes several bug and QOL fixes to the Scarab ship.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Comment by RustingWithYou** [comment]
!bugfix

File: html/changelogs/RustingWithYou - thegravityofthesituation.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixes a number of bugs with the Scarab ship."
```
Comment (FluffyGhoster) [inline html/changelogs/RustingWithYou - thegravityofthesituation.yml:41]: Need to list what is fixed

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

File: maps/away/ships/coc/coc_scarab/coc_scarab_ghostroles.dm [hunk 2]
```diff
@@ -38,6 +39,6 @@
 	max_count = 1
 	assigned_role = "Scarab Salvager Captain"
 	special_role = "Scarab Salvager Captain"
-	desc = "Command a salvage ship. Harvest valuable resources to take back to the wider Scarab Fleet. Characters in this role must have a name and background appropriate to the Scarab Fleet."
+	desc = "Command a salvage ship. Harvest valuable resources to take back to the wider Scarab Fleet. OOC Note: Characters must have names and backgrounds consistent with the Scarab Fleet. This is enforcable by admin action"
```
Comment (NonQueueingMatt) [inline maps/away/ships/coc/coc_scarab/coc_scarab_ghostroles.dm:42]:
```suggestion
	desc = "Command a salvage ship. Harvest valuable resources to take back to the wider Scarab Fleet. OOC Note: Characters must have names and backgrounds consistent with the Scarab Fleet. This is enforcable by admin action."
```

**Review by DreamySkrell — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**